### DISCUSSION TITLE
Discerning Touch Input Types for Standard Tool Operation

### DISCUSSION THREAD

**Discussion by TonyNRT** [pubpub-zz/ppInk#301] [Q&A]
@pubpub-zz 

Is it currently possible for ppink to discern the type of user input and implement a discrete tool respectively?

For example, the three main inputs I am concerned about are Single Finger, Stylus and Palm/Hand on a touch device. 

Ideally:
Single Finger Input =  Pointer Operation
Stylus Input = Current Pen Tool and Color
Palm/Hand = Erase Functionality

Additionally:
Multi-Point touch based on relative distance = zoom or multiple pen inputs
Mouse input = is always cursor mode

If this is not currently possible could you share any information on the feasibility of adding this feature in the future? 

I appreciate your assistance!

**Comment by pubpub-zz** [comment]
ppInk uses the old Microsoft.Ink library. It can not detect what you are looking for. Sorry.

**Reply by TonyNRT** [reply]
@pubpub-zz 
Would you be open to a live conversation regarding ppink? 

We are eager to continue with its implementation and would like to understand the solution in a bit more detail. Perhaps there could be some contract development work you would be willing to offer.

**Comment by pubpub-zz** [comment]
I'm sorry, I'm currently quite overbooked but maybe soon.
However Do not hesitate to review the code and ask questions in this thread.

about your last point about contract development, I have trouble to understand whay you mean : can you clarify?
